.travis.yml

```diff
@@ -24,7 +24,7 @@ matrix:
 
 install:
   - pip install -q --use-mirrors nose python-dateutil $NUMPY pep8==1.5.7 pyparsing pillow
-  - sudo apt-get update && sudo apt-get -qq install inkscape libav-tools mencoder
+  - sudo apt-get update && sudo apt-get -qq install inkscape libav-tools mencoder dvipng texlive-latex-base texlive-latex-extra texlive-fonts-recommended
```

Comment:
Perhaps there's no alternative, but I think all those tex packages are huge and will slow down the builds. The existing lines below indicate that the penalty might be reduced by using --no-install-recommends.